features/gift_exchanges/challenge_yuletide.feature

```diff
@@ -32,7 +32,8 @@ Feature: Collection
     And I add the fandom "Starsky & Hutch" to the character "Foo The Wonder Goat"
     And a character exists with name: "Obscure person", canonical: true
     And I add the fandom "Tiny fandom" to the character "Obscure person"
-  When I go to the collections page
+  When all indexing jobs have been run 
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.